### PR TITLE
Project ARES: Datacore

### PR DESCRIPTION
**Pull request by realforest2001**
Moves most information from ARES Interface and APOLLO Interface to the ARES Datacore. Also reorganises the files so the two interfaces are in separate files with all their UI data.
# About the pull request
Centralises most of the information used by the ARES Interface and APOLLO Maintenance Console(s) into a datacore.
Also reorganises the files so the two consoles are in their own individual files with all their UI data rather than having them separate from the UI data which was making it difficult to navigate.

Also adds checks to everywhere I could find missing them for logging to the datacore.

# Explain why it's good for the game
This will help massively for future compatability and updates.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
code: Moved most data storage from ARES Interface and APOLLO consoles to a new ARES datacore datum.
code: Moved the two consoles to separate files for easier navigation, and moved their respective UI data to the files.
code: Ares logging procs are now global procs with integrated can_log checks, rather than requiring definition of ares_link for every use.
add: Added ares_can_log checks to all the places I could find missing them.
fix: Fixed missing rejection button for access tickets.
del: Removed claim ticket button from Access Tickets.
/:cl:

### CODE REVIEW

File: code/modules/cm_marines/overwatch.dm [hunk 1]
```diff
@@ -798,7 +798,9 @@
 	log_attack("[key_name(user)] fired an orbital bombardment in [A.name] for squad '[current_squad]'")
 
 	/// Project ARES interface log.
-	GLOB.ares_link.log_ares_bombardment(user.name, ob_name, "X[x_bomb], Y[y_bomb] in [A.name]")
+	var/datum/ares_datacore/datacore = GLOB.ares_datacore
+	if(ares_can_log())
```
Comment (morrowwolf) [inline code/modules/cm_marines/overwatch.dm:802]: This should really be like inside the proc you are calling if it's required before every time you call the proc

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/game/machinery/computer/almayer_control.dm [hunk 2]
```diff
@@ -131,8 +130,8 @@
 
 			log_game("[key_name(usr)] has called for an emergency evacuation.")
 			message_admins("[key_name_admin(usr)] has called for an emergency evacuation.")
-			var/datum/ares_link/link = GLOB.ares_link
-			link.log_ares_security("Initiate Evacuation", "[usr] has called for an emergency evacuation.")
+			if(ares_can_log())
```
Comment (Zonespace27) [inline code/game/machinery/computer/almayer_control.dm:133]: for all of these, why can't this proc be inside the `log_ares_XXXX` procs?

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**